recipes-core/initrdscripts/files/ptest/test_ni_provisioning.sh

```diff
@@ -0,0 +1,176 @@
+#!/bin/bash
+#
+# Test that the booted storage disk matches the partition scheme which we
+# expect from the ni_provisioning script. This ptest only supports the efi-ab
+# bootscheme present on new x64, RAUC-based targets.
+exec 2>&1
+
+
+PTEST_TEST='ni_provisioning'
+
+print_error() {
```

Comment:
Non-blocking for this review: In the interest of parsing test output, can these functions be made available to future tests? This and the [OK], [FAIL], INFO logs.

Reply:
Yeah; we should write a standard ptest library at some poiint. I kind-of wrote one for the [kernel-tests](https://github.com/ni/meta-nilrt/blob/nilrt/master/dunfell/recipes-kernel/kernel-tests/kernel-tests-files/ptest-format.sh) packages, but didn't have the opportunity to make it into a shared library for other packages.

I intend on addressing that work in the later API guarantee features.